### PR TITLE
Nz/4304 Update Budget and FFP for MMIS

### LINKED ISSUE #4304: [Feature] Update Budget and FFP for MMIS
Describe the task here and link or add dependencies to any applicable issues...

MMIS APDs do not need the Estimated Quarterly Expenditure Table in the Budget and FFP page.

Figma link of how the Budget and FFP page should look after removing the EQE table:
https://www.figma.com/file/hJpKHKU6fz5J0Z7fisSwa2/eAPD-MMIS-2022?node-id=601%3A12578

Design ticket: #4243

Test cases: 
| Given  | When | Then |
| ------------- | ------------- | ------------- |
| A HITECH APD  | On the Budget and FFP page  | The EQE Table **should** be visible | 
| A MMIS APD  | On the Budget and FFP page  | The EQE Table **should not** be visible | 

### This task is done when...

- [ ] MMIS APDs do not have the EQE tables in Budget and FFP

### PR DESCRIPTION
**Pull request by Sun-Mountain**
Resolves #4304 

### Description
MMIS APDs do not need the Estimated Quarterly Expenditure Table in the Budget and FFP page.

### Significant changes or possible side effects
Figma link of how the Budget and FFP page should look after removing the EQE table:
https://www.figma.com/file/hJpKHKU6fz5J0Z7fisSwa2/eAPD-MMIS-2022?node-id=601%3A12578

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| A HITECH APD | On the Budget and FFP page | The EQE Table should be visible | cypress |
| A MMIS APD | On the Budget and FFP page | The EQE Table should **not** be visible | cypress |

### Steps to manually verify this change

1. Go to `Activity > Budget and FFP` in HITECH APD and confirm table is there.
2. Go to `Activity > Budget and FFP` in MMIS APD and confirm table is **NOT** there.

### This pull request is ready to code review when

- [x] Automated tests are updated (and all tests are passing)
- [x] New automated test cases are documented above
- [x] Pull request has been labeled, if applicable with feature, content, bug,
      tests, refactor
- [ ] Associated OpenAPI documentation has been updated
- [ ] The experience passes a basic manual accessibility audit (keyboard nav,
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when

- [x] Code has been reviewed by someone other than the original author

### This pull request is ready to review when the QA has

- [x] Verified the functionality related to the change
- [x] Verified that the change works with Narrator on Windows
- [x] Verified that the change works with VoiceOver on Mac
- [x] Verified all updated pages with the WAVE tool
- [x] Verified tab and keyboard navigation functionality

### This pull request can be merged when

- [ ] Design has approved the experience
- [x] Product has approved the experience

### CODE REVIEW

File: web/src/pages/apd/activities/ffp/CostAllocateFFP.js [hunk 1]
```diff
@@ -129,6 +132,7 @@ const CostAllocateFFP = ({
   });
 
   useEffect(() => {
+    console.log({ apdType });
```
Comment (thetif) [inline web/src/pages/apd/activities/ffp/CostAllocateFFP.js:135]: remove console log statement

**Review by thetif — APPROVED** [review]
great work!

File: web/src/pages/apd/activities/ffp/CostAllocateFFP.js (hunk 1 above)
Comment (Sun-Mountain) [reply web/src/pages/apd/activities/ffp/CostAllocateFFP.js:135]: doh

**Review by tbolt — APPROVED** [review]

File: web/src/pages/apd/activities/ffp/CostAllocateFFP.js [hunk 2]
```diff
@@ -396,11 +403,13 @@ const mapStateToProps = (
...
     activityId: activity.activityId,
     activityName: activity.name || 'Untitled',
+    apdType: state.apd.data.apdType,
     costAllocation: getCostAllocation(state, { activityIndex }),
     costSummary: getCostSummary(state, { activityIndex }),
     stateName: getState(state).name,
     otherFunding: activityTotal.data.otherFunding,
-    adminCheck: selectAdminCheckEnabled(state)
+    adminCheck: selectAdminCheckEnabled(state),
+    year: getAPDYearRange(state)
```
Comment (amyd11) [inline web/src/pages/apd/activities/ffp/CostAllocateFFP.js:412]: Good use of this reducer

**Review by amyd11 — APPROVED** [review]
Pretty straightforward & works as expected!

**Review by mirano-darren — APPROVED** [review]
Looks good to me!
| Given  | When | Then | Result | 
| ------------- | ------------- | ------------- | ------------- |
| A HITECH APD  | On the Budget and FFP page  | The EQE Table **should** be visible | ✅ |
| A MMIS APD  | On the Budget and FFP page  | The EQE Table **should not** be visible | ✅ |

**Review by jeromeleecms — APPROVED** [review]
LGTM!

**Comment by jeromeleecms** [comment]
Also adding @SGilliamA1M for design sign off.

**Review by SGilliamA1M — APPROVED** [review]
Changes look great!
- The EQE Table is not visible for MMIS APDs.
- The EQE Table is visible for Hitech APDs.
Great job!

**Review by thetif — APPROVED** [review]

**Pull request merged**